src/github.ts

```diff
@@ -134,5 +153,8 @@ export enum Conclusion {
     Failure = "failure",
     Neutral = "neutral",
     Cancelled = "cancelled",
-    TimedOut = "timed_out"
+    TimedOut = "timed_out",
+
+    // this sets the status of the check run to "in_progress"
```

Comment:
There is no "in_progress" conclusion, only a status.
To avoid handling two properties, we can infer the status from the conclusion - see https://github.com/brigadecore/brigade-github-app/blob/master/cmd/check-run/main.go#L77-L81